WooCommerce/Classes/View Modifiers/View+Conditionals.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+extension View {
+    /// Renders a view if the provided  `condition` is met.
+    /// If the `condition` is not met, an instance of `EmptyView` will be used in place of the receiver view.
+    ///
+    func renderedIf(_ condition: Bool) -> some View {
```

Comment:
Is it probably better to rename it to `renderIf(_ condition: Bool)`?

Reply:
I was looking at `SwiftUI` API naming and saw examples like "hidden()" instead of "hide()".
That's why I choose "renderedIf()" instead of "renderIf()"

Makes sense?

Reply:
Makes sense ;)